skt/templates/merge_details.j2

```diff
@@ -1,8 +1,8 @@
 
 We cloned this repository and checked out a ref:
 
-  Repo: {{ mergedata['base'][0] }}
-  Ref:  {{ mergedata['base'][1] }}
+  Repo: {{ mergedata['baserepo'] }}
+  Ref:  {{ mergedata['basehead'][:12] }} ({{ mergedata['basesubject'] }})
```

Comment:
Do you think you could remove the parentheses around the subject? This way we'd get less visual noise, and make it easier to read. This would also match the widely-recognized `git log --oneline` output.

Reply:
Oh, and I don't mean just this place in the template code where commit subject is pasted, but all of them, of course,

Reply:
Fixed!